_data/iot.yml

```
@@ -55,6 +55,7 @@ websites:
     img: ringcom.png
     tfa:
       - sms
+      - email
     doc: https://support.ring.com/hc/en-us/articles/360024818291
```

The old URL does not seem to be valid anymore.
```suggestion
    doc: https://support.ring.com/hc/en-us/articles/360039693891
```